src/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const OUTPUT_TEMPLATE_NAME = 'daily_output.base.json';
+export const OUTPUT_NAME = 'daily_output.json';
+export const ONE_GiB = 134_217_728;
```

Comment:
It should be
```suggestion
export const ONE_THOUSAND_MB = 1000 * 1000 * 1;
```

Reply:
[7a081bf](https://github.com/ardriveapp/inferno-backend/pull/8/commits/7a081bf5c0c00c0de33c4ee89766f819f59218e3)